server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -85,7 +85,7 @@
  * <b>table.custom.balancer.host.regex.max.outstanding.migrations</b>
  *
  */
-public class HostRegexTableLoadBalancer extends TableLoadBalancer implements ConfigurationObserver {
```

Comment:
Removing this interface could cause user code that assigns the variable to a type, or any subclasses that explicitly override those methods to fail to compile. I think it's probably low-risk enough that it's okay, but just wanted to raise the point for consideration.